nerdlandbot/commands/pets.py

```diff
@@ -0,0 +1,309 @@
+                return await ctx.send(message)
+
+        await ctx.send(message, file=discord.File(filepath))
+
+    @commands.command(
+        name="remove_pet",
+        aliases=["rm_pet"],
+        brief="remove_pet_brief",
+        usage="remove_pet_usage",
+        help="remove_pet_help",
+    )
+    @commands.guild_only()
+    async def remove_pet(
+        self, ctx: commands.Context, pet_id: typing.Optional[str] = None
+    ):
+        lang, guild_data = await asyncio.gather(
+            culture(ctx),
+            get_guild_data(ctx.message.guild.id),
+        )
+
+        if pet_id not in guild_data.pets:
+            message = translate("pet_picture_does_not_exist", lang).format(pet_id)
+            return await ctx.send(message)
+
+        if not (guild_data.user_is_admin(ctx.author) or str(ctx.author.id) == guild_data.pets[pet_id]["owner"]):
+            gif = translate("not_admin_gif", lang)
+            return await ctx.send(gif)
+
+        await guild_data.delete_pet(pet_id)
+        guild_path = Path.cwd() / PETS_DIR_NAME / str(ctx.guild.id)
+        filepath = guild_path / f"{pet_id}.jpg"
+        filepath.unlink()
+        message = translate("pet_removed", lang).format(pet_id)
+        await ctx.send(message)
+                
+
+    @commands.command(
+        name="add_category",
+        help="pet_add_category_help",
+        usage="pet_add_category_usage",
+        brief="pet_add_category_brief")
+    @commands.guild_only()
+    async def add_category(
+        self, ctx: commands.Context, category_name: typing.Optional[str] = None
+    ):
+        lang, guild_data = await asyncio.gather(
+            culture(ctx),
+            get_guild_data(ctx.message.guild.id),
+        )
+        if category_name is None:
+            msg = translate("pet_category_need_name", lang)
+            return await ctx.send(msg)
+
+        if not guild_data.user_is_admin(ctx.author):
+            gif = translate("not_admin_gif", lang)
+            return await ctx.send(gif)
+        
+        if await guild_data.add_new_pet_category(category_name.lower()):
+            msg = translate("pet_category_succes", lang).format(category_name)
+        else:
+            msg = translate("pet_category_error", lang)
+        return await ctx.send(msg)
+
+        
+
+    @commands.command(
+        name="show_categories",
+        brief="pet_show_categories_brief",
+        help="pet_show_categories_help"  
+    )
+    @commands.guild_only()
+    async def show_categories(self, ctx: commands.Context):
+        lang, guild_data = await asyncio.gather(
+            culture(ctx),
+            get_guild_data(ctx.message.guild.id),
+        )
+        if not guild_data.pets_categories:
+            msg = translate("pet_no_categories", lang)
+            return await ctx.send(msg)
+
+        msg = translate("pet_categories", lang)
+        for category in guild_data.pets_categories:
+            msg += f"\n -"
+            msg += category.capitalize()
```

Comment:
You're not using the fstring.
```suggestion
            msg += f"\n -{category.capitalize()}"
```